src/QMCWaveFunctions/lcao/SoaLocalizedBasisSet.h

```diff
@@ -93,11 +93,12 @@ struct SoaLocalizedBasisSet : public SoaBasisSetBase<ORBT>
   }
   /** set Number of periodic Images to evaluate the orbitals. 
       Set to 0 for non-PBC, and set manually in the input.
+      Passes the pre-computed phase factor for evaluation of complex wavefunction. If WF is real Phase_factor is real and equals 1.  
```

Comment:
It is better to be more accurate. Phase factor can be -1 when WF is real and non-gamma.

Reply:
Added